src/domain/landingPage/LandingPage.tsx

```diff
@@ -7,6 +7,7 @@ import {
   useLandingPagesQuery,
 } from '../../generated/graphql';
 import useLocale from '../../hooks/useLocale';
+import { useMobile } from '../../hooks/useMobile';
```

Comment:
why not just `import { useMobile } from 'hds-react';` 
instead of creating own useMobile hook?

Reply:
Only the typing is exported from the lib